visual-challenge/src/main/java/applitools/hackathon/entities/User.java

```diff
@@ -0,0 +1,12 @@
+package applitools.hackathon.entities;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import com.epam.jdi.tools.DataClass;
+
+public class User extends DataClass<User> {
+    public String username = "Roman Iovlev", password = "p@s$w0rd";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4513258)